stag-library-compiler/src/main/java/com/vimeo/stag/processor/utils/TypeUtils.java

```diff
@@ -170,6 +208,26 @@ public static boolean isConcreteType(@NotNull TypeMirror typeMirror) {
         return true;
     }
 
+
+    public static boolean containsTypeVarParams(@NotNull TypeMirror typeMirror) {
```

Comment:
could we get some javadocs on this method, as it would be the only public method without docs?